test/ecto_ldap_test.exs

```diff
@@ -185,4 +186,124 @@ defmodule EctoLdapTest do
     end
   end
 
+  test "loading :id type" do
```

Comment:
These need to be converted back to DocTests. That whole section of docs and examples provides valuable information for interacting with the adapter. People should be able to look at the docs for it, not dig around in tests.